crates/vizia_core/src/views/checkbox.rs

```diff
@@ -133,7 +133,7 @@ impl Checkbox {
     /// #
     /// Checkbox::new(cx, AppData::value);
     /// ```
-    pub fn new(cx: &mut Context, checked: impl Lens<Target = bool>) -> Handle<Self> {
+    pub fn new<'a>(cx: &'a mut Context, checked: &impl Lens<Target = bool>) -> Handle<'a, Self> {
```

Comment:
fyi: this is not necessary, lifetime elision can handle it, whatever linter is yelling at you about it is lying to you. clippy will yell about this form.